### DISCUSSION TITLE
Ozonsonde

### DISCUSSION THREAD

**Discussion by Pedritoprint** [dl9rdz/rdz_ttgo_sonde#333] [Q&A]
Till today, i never saw an "rdzTTGOsonde"-Station reporting the Ozon-values from the (RS41-SG) sonde on sondehub.org. Will these datas not be decoded in rdz-SW or not sent to Sondehub? 
greetings from Switzerland HB9CMI

**Comment by dl9rdz** [comment]
No, the ozon data is currently not being decoded.

**Reply by Pedritoprint** [reply]
Thanks
